### ISSUE TITLE
NTR: alpha-1,3-fucosyltransferase low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8351]
New term request alpha-1,3-fucosyltransferase low in blood (elthat_130605183825):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000512
- label: Reduced circulating alpha-1,3-fucosyltransferase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:9448836
- def: Concentration or activity of alpha-1,3-fucosyltransferase is below the lower limit of normal in the blood circulation.
- comment: The alpha 3 fucosyltransferases are a family of glycosyltransferases involved in the addition of fucose onto glycoproteins and glycolipids. One of the best defined roles for the alpha 3 fucosyltransferases is in the biosynthesis of the carbohydrate antigen sialyl Lewis X, the minimal ligand for the selectin family of adhesion molecules.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8351
- synonyms:
